pkg/projects/projects.go

```diff
@@ -0,0 +1,5 @@
+package projects
+
+func Exists(name string) bool {
+	return name == "01-snake-eater"
```

Comment:
I don't think this should get encoded into the binary.

Reply:
Ya, I didn't want to go too much into the projects api just yet.

Reply:
Fair.